pkg/metadata/metadata.go

```diff
@@ -126,6 +128,34 @@ func ApplyRebootAnnotation(ctx context.Context, c client.Client, node core.Node)
 	return ApplyLabelsAndAnnotations(ctx, c, node, nil, map[string]string{RebootAnnotation: ""})
 }
 
+// ApplyTlsAnnotation applies an annotation to the given secret communicating that the TLS certs need update
+func ApplyTlsAnnotation(ctx context.Context, c client.Client, secret *core.Secret) error {
```

Comment:
```suggestion
func ApplySecretTlsAnnotation(ctx context.Context, c client.Client, secret *core.Secret) error {
```

to avoid confusion with node' annotations helper functions.